src/icons/VisTypes.js

```diff
@@ -0,0 +1,883 @@
+import React from 'react';
+// import { colors } from '@dhis2/ui-core';
+import PropTypes from 'prop-types';
+import {
+    VIS_TYPE_COLUMN,
+    VIS_TYPE_STACKED_COLUMN,
+    VIS_TYPE_BAR,
+    VIS_TYPE_STACKED_BAR,
+    VIS_TYPE_LINE,
+    VIS_TYPE_AREA,
+    VIS_TYPE_PIE,
+    VIS_TYPE_RADAR,
+    VIS_TYPE_GAUGE,
+    VIS_TYPE_BUBBLE,
+    VIS_TYPE_YEAR_OVER_YEAR_LINE,
+    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
+    VIS_TYPE_SINGLE_VALUE,
+    VIS_TYPE_PIVOT_TABLE,
+} from '@dhis2/analytics';
+
+
+
+const VisIcon = ({ name, style }) => {
+    switch (name) {
+        case VIS_TYPE_COLUMN:
+            return (
+                <svg
```

Comment:
Can these icon definitions also be imported from `@dhis2/analytics`?
We now have the same icons in several places, including d2-ui `FavoritesDialog` component.

Reply:
Yepp!